src/Raven.Studio/wwwroot/App/views/database/tasks/editOlapEtlTask.html

```diff
@@ -54,7 +54,7 @@ <h3 data-bind="text: $parent.isAddingNewOlapEtlTask() ? 'New OLAP ETL' : 'Edit O
                             </div>
                             <div data-bind="validationElement: customPartition, collapse: customPartitionEnabled">
                                 <div class="form-group" data-bind="validationElement: customPartition">
-                                    <label for="taskName" class="control-label">Custom Partition</label>
+                                    <label for="taskName" class="control-label">Partition Value</label>
```

Comment:
Why this got changed as well? Is that correct? @aviv86?

Reply:
@aviv86 
Imho, writing `Partition Value` is more accurate than `Custom Partition`
It is the **value** that is entered there, not the partition name